README.md

```diff
@@ -16,161 +16,8 @@ Instructions for [Swift Package Manager](https://swift.org/package-manager/) sup
 found in the [SwiftPackageManager.md](SwiftPackageManager.md) Markdown file.
 
 ## Getting Started
-Firebase Data Connect is in Private Preview at no cost for a limited time. Sign up the program at https://firebase.google.com/products/data-connect.
-
-Once you are selected as an allowlist member, you should be able to create a Cloud SQL instance through Firebase Data Connect console.
-
 Here's a quick rundown of steps to get you started. Learn more about details at the official [Getting Started documentation](https://firebase.google.com/docs/data-connect/quickstart).
```

Comment:
Delete "Here's a quick rundown of steps to get you started. "

Reply:
Fixed. thanks for the catch